### ISSUE TITLE
Remove app css border when using native frame

### ISSUE THREAD

**Issue by KeitIG** [KeitIG/museeks#183]

**Comment by YurySolovyov** [comment]
Kinda need some evaluation first:
![image](https://cloud.githubusercontent.com/assets/2168518/18026789/4c689926-6c5a-11e6-8e16-21acd52ba62b.png)
![image](https://cloud.githubusercontent.com/assets/2168518/18026791/5d90e24e-6c5a-11e6-830f-37e77d556821.png)

**Comment by KeitIG** [comment]
Ah,

interesting. We'd better keep them for desktop environement without borders to their native frame

**Comment by YurySolovyov** [comment]
Yeah, this one was mint18 with native frame, I think we still need at least some borders

**Comment by KeitIG** [comment]
Indeed. Thanks for pointing it out.
